plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1093,6 +1109,9 @@ private Deployment createOpenShiftDeploymentInternal(String workspaceID,
                                     .withLivenessProbe(getLivenessProbeFrom(exposedPorts))
                                     .withCommand(command)
                                     .withVolumeMounts(getVolumeMountsFrom(volumes, workspaceID, withSubpath))
+                                    .withNewResources()
+                                        .withLimits(resourceLimits)
```

Comment:
It is shifted on purpose; the dsl for openshift-client has nested elements, so it's more readable to indent those elements to expose the structure. The only methods available between `withNewResources()` and `endResources()` are `withLimits()` and `withRequests()`. A more obvious example of this is a little further down, where the `Deployment` is created.